src/content/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence.mdx

```diff
@@ -11,7 +11,13 @@ redirects:
   - /docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-new-relic-ai
 ---
 
-With Applied Intelligence's Proactive Detection, anomalies from your [APM](/docs/apm)-monitored applications are automatically surfaced in our activity streams and anomalies feed. Each anomaly can be clicked on to bring up an automatic analysis. You can have notifications for anomalies delivered in Slack, or you can set up a webhook to deliver messages when you need them. These events are available for querying, creating custom dashboards, and alerting. By creating a Proactive Detection configuration (a group of apps you’re interested in), you can then add this configuration as a source, and have anomalies automatically correlated with other data sources via Incident Intelligence.
+With Applied Intelligence's Proactive Detection, anomalies from your APM-monitored applications are automatically surfaced in our activity streams and anomalies feed. Each anomaly can be clicked on to bring up an automatic analysis.
+
+You can have notifications for anomalies delivered in Slack, or you can set up a webhook to deliver messages when you need them. These events are available for querying, creating custom dashboards, and alerting. By creating a Proactive Detection configuration (a group of apps you’re interested in), you can then add this configuration as a source, and have anomalies automatically correlated with other data sources via Incident Intelligence.
```

Comment:
I'd change the second "creating". how about "By configuring Proactive Detection..."

Reply:
thx, more copyediting is in order!